x/twap/api_test.go

```diff
@@ -238,9 +237,156 @@ func (s *TestSuite) TestGetArithmeticTwap() {
 	for name, test := range tests {
 		s.Run(name, func() {
 			s.SetupTest()
-			for _, record := range test.recordsToSet {
-				s.twapkeeper.StoreNewRecord(s.Ctx, record)
```

Comment:
Note to reviewers: this is the only change in this test fixture (converting loop to using `preSetRecords` helper